os/android/iahwc2.cpp

```diff
@@ -395,10 +401,12 @@ HWC2::Error IAHWC2::HwcDisplay::AcceptDisplayChanges() {
 
 HWC2::Error IAHWC2::HwcDisplay::CreateLayer(hwc2_layer_t *layer) {
   supported(__func__);
+  layer_idx_ = display_.GenerateLayerHash(layer_idx_);
+  layer_idx_ += timestamp_;
   layers_.emplace(static_cast<hwc2_layer_t>(layer_idx_), IAHWC2::Hwc2Layer());
   layers_.at(layer_idx_).XTranslateCoordinates(display_->GetXTranslation());
   *layer = static_cast<hwc2_layer_t>(layer_idx_);
-  ++layer_idx_;
+  layer_idx_ = (uint64_t)&layers_.at(layer_idx_);
```

Comment:
just for clarity can u change this to (uint64_t)&(layers_.back().second) ?

Reply:
But maps don't seem to have a back() method? [API Doc](http://www.cplusplus.com/reference/map/map/)

Reply:
Sorry missed that.